app/api/available_room.py

```
@@ -17,19 +17,68 @@
+                            }
+                        }
+                    }
+                }
+            }
+        }
+    },
 )
 @router.get("", response_model=ApiResponse[AvailabilityResponse], include_in_schema=False)
 @limiter.limit(f"{RATE_LIMIT_PER_MINUTE}/minute")  # Rate Limit 적용
 async def check_room_availability(
     request: Request,
```

Ruff (0.15.1)

[warning] Unused function argument: `request`

(ARG001)